### ISSUE TITLE
Clarification on Group_FNs Labels in HCP_1200_surface_FN17.mat

### ISSUE THREAD

**Issue by albertleemon** [MLDataAnalytics/pNet#5]
Dear Experts,

I hope this message finds you well. I have a quick question regarding the predefined Group_FNs in the file hcp_1200_surface_FN17.mat.

Could you kindly confirm the labels or order of the 17 networks in this file? I assume the networks follow Yeo’s 17-network parcellation scheme, but I am aware that different versions of the network order exist.

Additionally, when looking at the generated plots for Group_FNs. 
![gFN_All(Compressed)](https://github.com/user-attachments/assets/ba6f9fd3-2947-4ad6-bfe2-21996b1cee6f)

I noticed that some networks do not seem to align with the order from the Schaefer labels, such as the one found in this [link](https://github.com/ThomasYeoLab/CBIG/blob/master/stable_projects/brain_parcellation/Schaefer2018_LocalGlobal/Parcellations/MNI/Centroid_coordinates/Schaefer2018_100Parcels_17Networks_order_FSLMNI152_1mm.Centroid_RAS.csv). For example, as showed in the figure, FN 9 should not be Limbic_B, while FN 10, which seems like the combination of Limbic BA and A.  What's the proper label of FN 14 and FN 17? 

N1: VisCent-Visual A (22 ROIs); 
N2: VisPeri-Visual B (18 ROIs); 
N3: SomMotA-Somatomotor A (30 ROIs); 
N4: SomMotB-Somatomotor B (21 ROIs); 
N5: DorsAttnA-Dorsal Attention A (16 ROIs); 
N6: DorsAttnB-Dorsal Attention B (16 ROIs); 
N7: SalVentAttnA-Salience/Ventral Attention A (25 ROIs); 
N8: SalVentAttnB-Salience/Ventral Attention B (14 ROIs); 
N9: LimbicB-Limbic B (9 ROIs); 
N10: LimbicA-Limbic A (11 ROIs); 
N11: ContA-Control A (22 ROIs); 
N12: ContB-Control B (18 ROIs); 
N13: ContC-Control C (8 ROIs); 
N14: DefaultA-Default A (23 ROIs); 
N15: DefaultB-Default B (25 ROIs); 
N16: DefaultC-Default C (10 ROIs); 
and N17: TempPar-Temporal Parietal (12 ROIs). 

I would greatly appreciate any clarification on the correct labeling of these 17 networks. Thank you very much in advance for your help!

Yours sincerely
Meng

**Comment by MLDataAnalytics** [comment]
These FNs were computed based on the young adult HCP data. They do not necessarily align with other atlases.

**Comment by albertleemon** [comment]
Thank you very much for your prompt response. I sincerely appreciate the excellent work you've done. I’m writing to seek clarification regarding the predefined Group_FNs file, as this could be a point of critique by reviewers, even though this current predefined version might be the best option available for most users.

Based on the publications listed on the homepage, I understand that several variations of group FNs have been developed, incorporating labels from both Gordon’s and Yeo’s parcellation schemes. Even when the same number of networks is used, the specific labels and network assignments seem to vary across studies. Initially, I thought I could follow the labels described in Cui's Neuron paper, but I realized he did not use the HCP cohort to generate the Group FN. Then I think it is similar to the one in the latest paper on 'Self-supervised deep learning predicts personalized functional networks,' but it is not identical. Additionally, I came across the Midbrain Atlas, which uses only 14 networks, further complicating the comparisons. 

While I understand that data-driven atlases can differ from predefined ones, the 17-network structure appears to be designed for comparison with well-established networks, such as those described by Yeo et al. (2011). I've noticed that comparisons with Yeo's networks have been consistently made in the listed publications. Therefore, I’m particularly curious about how these networks are labeled on your side with the current version. 
 **Would it be possible to provide a text file with each network's annotations and the relevant citation?**

I’m seeking clarification on the labeling and creation process for these 17 networks because I’m eager to apply the predefined Group_FNs and ensure I fully understand the details and suitability for my work.

Thank you again for your help, 
Yours sincerely
Meng
